pkg/virt-api/rest/subresource.go

```diff
@@ -83,6 +84,7 @@ type SubresourceAPIApp struct {
 	statusUpdater           *status.VMStatusUpdater
 	clusterConfig           *virtconfig.ClusterConfig
 	instancetypeMethods     instancetype.Methods
+	httpClient              *http.Client
```

Comment:
nit: I would name it `handlerHttpClient`, similar to `handlerTLSConfiguration`.

Reply:
:+1:
Done